doc/releases/release-notes-3.6.rst

```diff
@@ -870,6 +870,29 @@ Libraries / Subsystems
 
 * File systems
 
+* Logging
+
+  * Added option to remove string literals from the binary when dictionary based logging is used.
+
+  * Optimized the most common logging messages (strings with up to 2 numeric arguments). Optimization
+    is done for code size (significant gain seen on riscv32) and performance.
+
+  * Extended logging frontend API to optionally implement dedicated functions for optimized messages.
+    Optional API is enabled by :kconfig:option:`LOG_FRONTEND_OPT_API`.
```

Comment:
I think it needs the `CONFIG_` prefix to link properly?

```suggestion
    Optional API is enabled by :kconfig:option:`CONFIG_LOG_FRONTEND_OPT_API`.
```

Reply:
fixed